cli/package.json

```diff
@@ -11,7 +11,7 @@
     "lint": "eslint .",
     "prepublish": "mkdir -p dist; npm run test",
     "test": "npm run clean; npm run build; npm run test-quick",
-    "test-quick": "npm run lint && jest",
+    "test-quick": "npm run lint; jest",
```

Comment:
Ah didn't mean to check this in, thanks.

It's annoying that `test-quick` won't run tests if there are lint errors, but the `&&` is necessary to make travis fail on lint errors (barring more complex bashisms)